internal/proto/token/v1/token.proto

```diff
@@ -13,6 +13,7 @@ service TokenService {
 
 message IssueReq {
   string user_id = 1;
+  uint32 user_role = 2;
```

Comment:
Why do we need `user_role` in `IssueReq` ?

Reply:
This is to send the user role to auth when creating the token.

Reply:
DomainUserID means id of user within the domain, which is different from actual user id for platform ,
In each domain, user should have unquie id .
So just we are adding prefix domain_id to user id to get the unqiue user id for user per domain , which is called domain user id